manifests/opa.yaml

```diff
@@ -152,7 +158,30 @@ spec:
         - name: opa-server
           secret:
             secretName: opa-server
-
+        - name: opa-config
+          configMap:
+            items:
+              - key: config.yaml
+                path: config.yaml
+            name: opa-config
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: opa-config
+  namespace: opa
+data:
+  config.yaml: |
+    services:
+      - name: {{.opa.bundle_service_name}}
```

Comment:
This is going to create an invalid config when the bundles are not set, we will need to add a check if the bundles are provided and adjust the config accordingly + add another test fixture without the bundle to verify it